tests/Integration/WhatsAppCloudApiTest.php

```diff
@@ -278,6 +278,26 @@ public function test_send_cta_url()
         $this->assertEquals(false, $response->isError());
     }
 
+    public function test_send_catalog_message()
+    {
+        $this->markTestIncomplete(
```

Comment:
same https://github.com/netflie/whatsapp-cloud-api/pull/190#discussion_r1662970563

Reply:
https://github.com/derrickobedgiu1/whatsapp-cloud-api/commit/a7ead6c6e1da75da6944b5c06acd4e8a679e7d6d

Removed optional user preferred `$sku_thumbnail` in test. README shows how the user can use it if they want